ui/packages/app/src/project_setting/secret/SubmitSecretForm.js

```diff
@@ -13,20 +13,34 @@ import {
   EuiIcon,
   EuiToolTip,
   EuiSpacer,
-  EuiPanel
+  EuiPanel,
+  EuiSelect
 } from "@elastic/eui";
 import { addToast, useMlpApi } from "@caraml-dev/ui-lib";
 import {
-  validateSecretKey,
+  validateSecretData,
   validateSecretName
 } from "../../validation/validation";
 
 const SubmitSecretForm = ({ projectId, fetchUpdates, secret, toggleAdd }) => {
+  const DEFAULT_SECRET_STORAGE_ID = 2;
```

Comment:
Should we add the user docs around the Secrets feature to this repo and remote sync to https://github.com/caraml-dev/docs ?

Reply:
Yes, I'll add this to the backlog first.